package.json

```diff
@@ -19,6 +19,7 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "@types/json-stable-stringify": "^1.0.32",
+    "draco-core": "^0.0.6",
```

Comment:
Why do we add a new dependency?

Reply:
It should be there... not sure why it was missing in the first place.